jscomp/build_tests/monorepo/apps/app/src/main.ml

```diff
@@ -0,0 +1 @@
+let result = NumberAdder.addNumbers 1 4
```

Comment:
can you commit the `JS` files (via in-source) to make sure imports are correctly emitted?

Reply:
Yes, thanks for the suggstion, I've committed the files 🙏